package.json

```diff
@@ -117,11 +127,12 @@
     "nx": "16.3.1",
     "nx-cloud": "latest",
     "persistgraphql": "^0.3.11",
-    "postcss": "^8.4.14",
+    "postcss": "^8.4.24",
     "react-test-renderer": "^18.2.0",
     "stylelint": "^15.10.1",
     "stylelint-config-prettier": "^9.0.3",
     "stylelint-config-recommended-scss": "^12.0.0",
+    "tailwindcss": "^3.3.2",
```

Comment:
This should be removed from all packages/*/package.json now that it's added to the root. You can do that with `yarn remove tailwindcss --all`, then `yarn add tailwindcss`